apps/server/src/modules/class/entity/testing/factory/class.entity.factory.ts

```diff
@@ -21,7 +21,7 @@ export const classEntityFactory = ClassEntityFactory.define(ClassEntity, ({ sequ
 		teacherIds: [new ObjectId(), new ObjectId()],
 		invitationLink: `link-${sequence}`,
 		year: new ObjectId(),
-		gradeLevel: sequence,
```

Comment:
Not using value objects or the production factory bring us always in the situation that we have no validation checks. 
gradeLevel 25 is allow to set for test factory, but should not. 

But make it fix to 3 is much better then sequence.

Reply:
a random int should be used. But we have no place with such utils so far